apps/emqx_web_hook/src/emqx_web_hook.appup.src

```diff
@@ -12,10 +12,23 @@
      {load_module,emqx_web_hook,brutal_purge,soft_purge,[]},
      {load_module,emqx_web_hook_actions,brutal_purge,soft_purge,[]}]},
    {"4.3.8",
-    [{load_module,emqx_web_hook_app,brutal_purge,soft_purge,[]},
+    [{load_module,emqx_web_hook_actions,brutal_purge,soft_purge,[]},
+     {load_module,emqx_web_hook_app,brutal_purge,soft_purge,[]},
+     {load_module,emqx_web_hook,brutal_purge,soft_purge,[]}]},
+   {"4.3.9",
+    [{load_module,emqx_web_hook_actions,brutal_purge,soft_purge,[]},
+     {load_module,emqx_web_hook,brutal_purge,soft_purge,[]},
+     {load_module,emqx_web_hook_app,brutal_purge,soft_purge,[]}]},
+   {"4.3.10",
+    [{load_module,emqx_web_hook_actions,brutal_purge,soft_purge,[]},
+     {load_module,emqx_web_hook,brutal_purge,soft_purge,[]},
+     {load_module,emqx_web_hook_app,brutal_purge,soft_purge,[]}]},
+   {"4.3.11",
+    [{load_module,emqx_web_hook_actions,brutal_purge,soft_purge,[]},
+     {load_module,emqx_web_hook,brutal_purge,soft_purge,[]}]},
+   {"4.3.12",
```

Comment:
The current version of the `emqx-webhook` application is 4.3.12, so we only need the commands for 4.3.0 to 4.3.11.

So anything related to 4.3.12 should be removed here.